portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -29,26 +30,26 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
 /** This servlet handles comment's data: 
  *- doPost() stores get's the recently submitted comment's data from the request 
  *          and stores in a database using Datastore API
  *- doGet() returns the comments from the database, aftern converting them to JSON*/
 @WebServlet("/comments-data")
 public class CommentsServlet extends HttpServlet {
-    private static final String COMMENT_ENTITY_NAME = "Comment";
-    private static final String FIRST_NAME_PROPERTY = "firstName";
-    private static final String LAST_NAME_PROPERTY = "lastName";
-    private static final String EMAIL_PROPERTY = "email";
-    private static final String MESSAGE_PROPERTY = "message";
-    private static final String PHONE_PROPERTY = "phone";
-    private static final String JOB_TITLE_PROPERTY = "jobTitle";
-
-
+    
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        Comment newComment = getCommentFromRequest(request);
-        putCommentToDatastore(newComment);
+        String userId = User.getUserIdFromUserService();
+        //it's guaranteed that the user is logged in at this point
```

Comment:
nits: First word of comment should have space before and capitalized

// It's guaranteed ...

Reply:
done